Clarify Secondary DNS and Let's Encrypt incompatibility (#1214)
dnsimple/dnsimple-support: content/articles/letsencrypt.md

The DNSimple Let's Encrypt integration allows you to request an SSL certificate for free using the Let's Encrypt certification authority.

<note>

> To request an SSL certificate with Let's Encrypt, the domains **must be delegated and resolving only with DNSimple**. The domain doesn't need to be registered with DNSimple. This means it is not compatible with Secondary DNS.

Reviewer: ```suggestion
To request an SSL certificate with Let's Encrypt, the domains **must be delegated and resolved exclusively with DNSimple**. The domain doesn't need to be registered with DNSimple, which means it is not compatible with Secondary DNS.
```

Author: I merged this with the other suggestions, thanks!

> To request an SSL certificate with Let's Encrypt, the domains **must be delegated to and exclusively resolving with DNSimple**, it is not compatible with Secondary DNS. The domain doesn't need to be registered with DNSimple.

Reviewer 2: This is a comma splice now, you need a conjunction or a period. 
 
```suggestion
To request an SSL certificate with Let's Encrypt, the domains **must be delegated to and exclusively resolving with DNSimple**. It is not compatible with Secondary DNS. The domain doesn't need to be registered with DNSimple.
```

Author: thank you so much
